meta-genivi-dev/recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -112,10 +112,9 @@ SRC_URI[index.sha256sum] = "86114b93f1f51aaf0aec3af0751d214b351f4ff9839ba031315c
 
 SYSTEMD_SERVICE_${PN} = "sota_client.service"
 
-DEPENDS += " openssl "
+DEPENDS += " openssl dbus "
```

Comment:
RVI SOTA client is written in the Rust programming language therefore it depends on Yocto/OE layer meta-rust which provides Rust and Cargo. Crate dbus is required by RVI SOTA Client and it is related to the Yocto/OE recipe for dbus. While I was working on the update of meta-rust and RVI SOTA Client, I noticed that in some cases the build may fail if dbus is not present at DEPENDS therefore I moved it from RDEPENDS to DEPENDS. 

Best regards,
Leon